conan/tools/cmake/layout.py

```diff
@@ -27,7 +27,12 @@ def cmake_layout(conanfile, generator=None):
         conanfile.folders.build = "cmake-build-{}".format(build_type)
         conanfile.folders.generators = os.path.join(conanfile.folders.build, "conan")
 
-    conanfile.cpp.source.includedirs = ["src"]
+    if src_folder == ".":
+        # The CMakeLists.txt is in the root, the includes (and sources) typically don't
```

Comment:
This is a bit confusing, maybe this means ``includedirs`` is yet another argument ``cmake_layout(..., include_dirs=xxx``? Or that should be an after ``cmake_layout()`` call, directly calling ``self.cpp.source.includedirs = ["xxx"]`` to change it? Why ``src_folder`` yes and ``includedirs`` no?

Reply:
That's why I added the comment, it is very confusing and quite random to provide "src"

Reply:
Lets make:
```python
conanfile.cpp.source.includedirs = ["include"]
```
For all cases

Reply:
done